.github/workflows/nodejs.yml

```diff
@@ -13,7 +13,15 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: 12.x
-    - name: npm install, build, and test
+    - name: django-s3-file-field-client npm install, build, and test
+      working-directory: ./django-s3-file-field-client
+      run: |
+        npm ci
+        npm run build --if-present
```

Comment:
Why `--if-present`? Should the build script ever not exist?

Reply:
Courtesy of Sam Graetzl